### ISSUE TITLE
[NEW FEATURE] Add lyrics

### ISSUE THREAD

**Issue by kostiantynlelikov** [Veetaha/veebot-discord#1]
Please add an option to setup bot to send lyrics for the current playing song

**Comment by akrava** [comment]
Yeah, pls add this feature, it will be the great enhancement for your bot :)
